src/Sulu/Component/SmartContent/ContentType.php

```diff
@@ -342,4 +347,22 @@ private function getCurrentPage($pageParameter)
 
         return intval($page);
     }
+
+    /**
+     * determine tags from current request.
+     *
+     * @param string $tagsParameter
+     *
+     * @return string[]
+     */
+    private function getTags($tagsParameter)
+    {
+        if ($this->requestStack->getCurrentRequest() !== null) {
+            $tags = $this->requestStack->getCurrentRequest()->get($tagsParameter, '');
+        } else {
+            $tags = '';
+        }
+
+        return array_filter(explode(',', $tags));
+    }
```

Comment:
Isn't that something that should go into an own service together with the stuff from the `TagTwigExtension`? So that both classes, this one and the `TagTwigExtension`, can use this service and we have less code duplication?